plugins/inputs/unbound/README.md

```diff
@@ -0,0 +1,186 @@
+# Unbound Input Plugin
+
+This plugin gathers stats from [Unbound - a validating, recursive, and caching DNS resolver](https://www.unbound.net/)
+
+### Configuration:
+
+```toml
+ # A plugin to collect stats from Unbound - a validating, recursive, and caching DNS resolver
+ [[inputs.unbound]]
+   ## If running as a restricted user you can prepend sudo for additional access:
+   #use_sudo = false
+
+   ## The default location of the unbound-control binary can be overridden with:
+   binary = "/usr/sbin/unbound-control"
+
+   ## By default, telegraf gathers stats for 3 metric points.
+   ## Setting stats will override the defaults shown below.
+   ## stats may also be set to ["all"], which will collect all stats
+   stats = ["total.*", "num.*","time.up", "mem.*"]
+```
+
+### Measurements & Fields:
+
+This is the full list of stats provided by unbound. Stats will be grouped by their prefix (eg thread0,
+total, etc). In the output, the prefix will be used as a tag, and removed from field names. See
+https://www.unbound.net/documentation/unbound-control.html for details.
+
+- unbound
+    thread0.num.queries
```

Comment:
I think we should consider replace dots with underscores the field `num_queries` so the fields fit style wise with the other plugins.